sdc/io/csv_ext.py

```diff
@@ -390,7 +390,9 @@ def to_varname(string):
     Replaces unavailable symbols with _ and insert _ if string starts with digit.
     """
     import re
-    return re.sub(r'\W|^(?=\d)','_', string)
+    var_name = re.sub(r'\W|^(?=\d)', '_', string)
+# handle underscores
+    return '_'.join(i for i in var_name.split('_') if i)
```

Comment:
What is the use case for it?

Reply:
If you want to replace all multiple _ with single _ then, please, update the docstring too.